tests/utils.test.ts

```
@@ -0,0 +1,120 @@
+import { getTextUptilCaretInElement } from '../src/utils/get-text-up-until-caret-in-element'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)